builds/checkin/edgelet.yaml

```diff
@@ -114,3 +114,53 @@ jobs:
         displayName: Format Code
       - bash: edgelet/build/linux/clippy.sh
         displayName: Clippy
+
+################################################################################
+  - job: code_coverage
+################################################################################
+    displayName: Code Coverage
+    variables:
+      coverage.excludes: "docker-rs*"         # comma separated exclude patterns
```

Comment:
Also ignore `external-provisioning`, `management` and `workload` ?

Reply:
I'm going to leave those because we should get coverage on them. The docker crate is a problem because we don't use so much of it